pkg/controller/worker/machines.go

```diff
@@ -234,7 +238,7 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 				MaxUnavailable: worker.DistributePositiveIntOrPercent(zoneIdx, pool.MaxUnavailable, zoneLen, pool.Minimum),
 				// TODO: remove the csi topology label when AWS CSI driver stops using the aws csi topology key - https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/899
 				// add aws csi driver topology label if it's not specified
-				Labels:                       utils.MergeStringMaps(pool.Labels, map[string]string{awsCSIDriverTopologyKey: zone}),
+				Labels:                       utils.MergeStringMaps(pool.Labels, map[string]string{AwsCSIDriverTopologyKey: zone}),
```

Comment:
Can you remove the comment ? I don't think this is applicable, although we have to use a topology label ourselves still.